tools/Utilities/IfeoTool/ImageFileExecutionOptions.cs

```diff
@@ -0,0 +1,197 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using DevHome.Common;
+using DevHome.IfeoTool.TelemetryEvents;
+using DevHome.Telemetry;
+using Microsoft.Win32;
+
+namespace DevHome.IfeoTool;
+
+[Flags]
+public enum IfeoFlags : uint
```

Comment:
can you add a link to documentation in a comment on where these flags came from?

Reply:
And I'm going to assume these couldn't be generated by CsWin32?

Reply:
> can you add a link to documentation in a comment on where these flags came from?

I'll try.  The problem is that we don't have good documentation here.  The only way to find them is to follow the breadcrumbs in the header file, after getting there for Application Verifier.  I'll see what I can do.

Reply:
> And I'm going to assume these couldn't be generated by CsWin32?

There are a number of downsides to using the CsWin32 variations of these, and I found the benefits of having them be a strongly typed flags enum to outweigh the convenience of importing the basic flags through CsWin32.

Reply:
fixed in latest iteration.